docs/Span.md

```diff
@@ -245,7 +245,7 @@ We need to enable the existing array bounds check optimizations for Span – in
 Span\<T\> will support reinterpret cast conversions to Span\<byte\>. It will also support unsafe casts between arbitrary primitive types. The reason for this limitation is that some processors don’t support efficient unaligned memory access.
 
 ##Platform Support Plans
-We want to ship Span\<T\> as a nugget fat package available for .NET Standard 1.1 and above. Runtimes that support by-ref fields and returns will get the fast two filed span. Other runtimes will get the slower three-field span. 
+We want to ship Span\<T\> as a nuget heritage package available for .NET Standard 1.1 and above. Runtimes that support by-ref fields and returns will get the fast two filed span. Other runtimes will get the slower three-field span. 
```

Comment:
We informally call them fat packages. Publicly, there is no name for that and I'm convinced we need one. The nature of packages allows having multiple assets per platform.